src/main/resources/logback-spring.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 
-  <property name="LOGS" value="/usr/local/var/lib/radar/appserver/logs" />
+  <property name="LOGS" value="logs" />
```

Comment:
Again don't commit changes that you made for local dev. This path is used in Linux and Docker containers.